.vscode/settings.json

```diff
@@ -1,14 +1,22 @@
 {
+    "files.associations": {
```

Comment:
Question, do you know if this is something that we could get included in VSCode's defaults?

Reply:
i've no idea, but i would guess not.
they hard-code an exception for their own files (e.g. settings.json, tsconfig.json), but if i were microsoft i wouldn't want to maintain a hard-coded mapping of community things